packages/test-sync/src/androidInstrumentedTest/kotlin/io/realm/kotlin/test/common

```diff
@@ -0,0 +1 @@
+./../../../../../../commonTest/kotlin/io/realm/kotlin/test
```

Comment:
Why do we both have a `common` link and a `shared` package 🤔

Reply:
Fixed